portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,13 @@
  * Adds comments from /data
  */
 async function getComments() {
-    const commentsData = await fetch("/data");
-    const comments = await commentsData.text();
-    document.getElementById("comments-container").innerHTML = comments;
+  const commentsData = await fetch("/data");
```

Comment:
Please use single quotes for all string literals https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes